sci-calculators/bc-gh/metadata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<name>Gavin D. Howard</name>
+		<email>yzena.tech@gmail.com</email>
+	</maintainer>
```

Comment:
You need to add a project or a developer to commit for you. It's usually the proxy-maint team, check from other proxy-maintained ebuilds for the correct syntax.

Reply:
Added proxy-maint team.